test/index.js

```diff
@@ -1,6 +1,6 @@
-const assert = require( 'assert' );
-const sinon = require( 'sinon' );
-const memoize = require( '../' );
+import assert from 'assert';
+import sinon from 'sinon';
+import memoize from '../index'
```

Comment:
This was just a test, I'm not getting any ESLint errors about semis anywhere, I tried adding `prettier` as a dep and a `.prettierrc`, but it's still not working.